packages/opentelemetry-metrics/test/Batcher.test.ts

```diff
@@ -24,7 +24,7 @@ describe('Batcher', () => {
     let meter: Meter;
     let fooCounter: types.BoundCounter;
     let barCounter: types.BoundCounter;
-    let counter: types.Metric<types.BoundCounter>;
+    let counter: types.Counter;
```

Comment:
I think we should change `types.` to `api.` everywhere in the project, this is just to be more consistent and avoid any confusion for new users. Perhaps this is not needed in this PR but in the folloup PRs.

Reply:
I'll open a follow up PR for this :)